src/shogun/util/factory.h

```diff
@@ -312,9 +295,7 @@ namespace shogun
 	 */
 	CPipelineBuilder* pipeline()
 	{
-		auto result = new CPipelineBuilder();
```

Comment:
this is also probably material for another PR, not this one :)

Reply:
yup, I just needed to fix this because it was causing "false positive" memory leaks